packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -8,9 +8,18 @@ import {
     ITaggedTelemetryPropertyType,
     ITelemetryProperties,
 } from "@fluidframework/common-definitions";
+import {
+    IFluidErrorBase,
+    isFluidError,
+    isValidLegacyError,
+} from "./fluidErrorBase";
+import {
+    mixinTelemetryProps,
+    copyProps,
+} from "./errorLoggingInternalHelpers";
 
 /** @returns true if value is an object but neither null nor an array */
-const isRegularObject = (value: any): boolean => {
+export const isRegularObject = (value: any): boolean => {
```

Comment:
can this move to errorLoggingInternalHelpers.ts  so it's not exported out of the package